plugin/trino-loki/src/test/resources/compose.yaml

```diff
@@ -0,0 +1,21 @@
+version: '3.4'
+services:
+    loki:
+        image: grafana/loki:3.1.0
+        depends_on:
+            - minio
+        ports:
+            - 3100:3100
```

Comment:
This would use a fixed port right ? Won't it cause additional issues if the port is already mapped ? I think we need to avoid situations like this. Do we need this minio or can we add it as a follow-up ?